cappy.py

```diff
@@ -80,22 +91,24 @@ def receive(self, data):
                 # result. In particular, we probably can't do _all_ of the
                 # unflattening in one step.
 
-                start_index = i+1
+                start_index = i + 1
         self.buf += data[start_index:]
         return objects
 
 
 class Protocol(object):
-    """Network protocol"""
+    """Network protocol."""
```

Comment:
I'm always puzzled by this. There's no complete sentence here, so should there be a period?